### DISCUSSION TITLE
I also noticed a small issue. Every time I play a video in the WSABuilds emulator, there’s some noise at the beginning during the first playback. It disappears after a moment, but the noise returns every time I open and play a video.

### DISCUSSION THREAD

**Discussion by amralden** [MustardChef/WSABuilds#479] [General]
Hello, MustardChef,

I also noticed a small issue. Every time I play a video in the WSABuilds emulator, there’s some noise at the beginning during the first playback. It disappears after a moment, but the noise returns every time I open and play a video.

I hope this can be resolved in future updates. Thank you again for your dedication, and I look forward to hearing from you.

**Comment by MustardChef** [comment]
Could you provide a screen recording of the issue?
Thank you.

**Reply by amralden** [reply]
I didn't know what the problem was exactly. It wasn't any error code. Every time I play the video clip, noise appears and disappears. Then every time I play the video clip, the noise comes back and disappears.

**Reply by amralden** [reply]
Will there be updates coming in the future that will solve these problems that did not appear because I was not able to explain them to you?

**Reply by amralden** [reply]
do you it will be fixed automatically with the upcoming updates With the stability it provides us

Am I not worried about this issue?

**Reply by amralden** [reply]
do you In upcoming updates yes or no

Don't leave me please ❤🌹🌹

**Reply by amralden** [reply]
I know you are busy, I am waiting for everything new from you on this emulator. Greetings from Egypt. ❤❤❤❤
